airbyte-webapp/src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,27 @@
+@use "../../scss/colors";
+@use "../../scss/variables";
+
+@keyframes fadeIn {
+  from {
+    opacity: 0;
+  }
+}
+
+.modal {
+  animation: fadeIn 0.2s ease-out;
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background: rgba(colors.$black, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 100;
+}
+
+.card {
+  margin-left: 93px;
+  max-width: calc(100% - 93px - variables.$spacing-lg * 2);
```

Comment:
Should we maybe set a smaller max-width here (e.g. the 585px we use also in the confirmation modal)? This feels like potentially very wide modals otherwise?

Reply:
I was incidentally setting a max-width in my PR as well and used that width: https://github.com/airbytehq/airbyte/pull/14634/files#diff-f42082ba017fd6e4b93d656564fa5fd4858972445eb56b27369eef535931a483R1-R3 since it otherwise looked very stretched filling the whole 2k monitor :D